spec/graphql/types/query_type_spec.rb

```diff
@@ -1,32 +1,132 @@
 describe Types::QueryType do
 
-  context 'MediaEntryType' do
+  describe 'MediaEntryType fields' do
 
-    subject { Types::MediaEntryType }
+    describe Types::QueryType.fields['mediaEntry'] do
 
-    it 'requires and "id" argument of ID type' do
-      id_argument = Types::QueryType.fields['mediaEntry'].arguments['id']
+      context 'field definition' do
 
-      expect(id_argument.type.class).to be(GraphQL::Schema::NonNull)
-      expect(id_argument.type.of_type).to be(GraphQL::Types::ID)
+        it 'requires and "id" argument of ID type' do
+          id_arg = subject.arguments['id']
+
+          expect(id_arg.type.class).to be(GraphQL::Schema::NonNull)
+          expect(id_arg.type.of_type).to be(GraphQL::Types::ID)
+        end
+      end
+
+      context 'response' do
+
+        let(:media_entry) { FactoryGirl.create(:media_entry_with_title) }
+        let(:query) { media_entry_query(media_entry.id) }
+        let(:response_data) { response_as_hash(query)[:data][:mediaEntry] }
```

Comment:
@kooropatfa You're calling this method over and over again along with `[:data]`, why just not to move it to the method's body?